apps/state/config/config.exs

```diff
@@ -195,6 +197,12 @@ config :state, :stops_on_route,
     ],
     {"CR-Newburyport", 0} => [
       [
+        "Beverly",
```

Comment:
These blocks in `stop_order_overrides` help ensure that the API returns the stops on `CR-Newburyport` in the order expected on the MBTA.com timetable. Before this change. Weekends (with the entire Rockport branch shuttled) in the new schedule, the Rockport branch stops would show up in a weird place on the timetable sometimes.